  # task_frame_to_PSM1_frame Always needed and always constant
  # task_frame_to_PSM2_frame Always needed and always constant
  # task_frame_to_right_cam_frame Used only if right cam pose is not provided online
  # task_frame_to_left_cam_frameUsed only if left cam pose is not provided online
  # left_cam_frame_to_right_cam_frameUsed when one of the cam poses are is available online
  # left_cam_frame_to_right_cam_frame: [-0.0, -0.0, -0.0, -0.0, 0.0, -0.0, 0.0]

  world_frame_to_PSM1_frame: [0.06141741621604557, -0.17984621739888426, -0.13080632073042162,
     -0.04750117023002374, -0.07554588580155144, 0.0010713328942994333, 0.9960096938338839]
  world_frame_to_PSM2_frame: [-0.22828661988738014, -0.16970579270235275, -0.10694326872425357,
     0.005708214995440446, 0.03405966733575918, -0.015576925008257256, 0.9992820996844836]
  world_frame_to_left_cam_frame: [0.0792294774569, -0.0625801545724, 0.27141894149,
    -0.028709680692, 0.985893665499, -0.104818263216, -0.127289301434]
  world_frame_to_right_cam_frame: [0.0792294774569, -0.0625801545724, 0.27141894149,
    -0.028709680692, 0.985893665499, -0.104818263216, -0.127289301434]
